client/.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_AUTH0_DOMAIN=dev-ynlvngvzhos0c4m0.us.auth0.com
```

Comment:
You should add /.evn in .gitignore, uploading them on github is not a good approach.

Reply:
sorry I had forgotten to add. Now I have added it.